portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Me</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <style>
+    a    {
+        color: black;
+        text-shadow: 2px 2px 5px turquoise, 1px 1px 3px purple, 1px 2px 3px snow;
+        text-decoration: underline;
+    }
+    body {
+        background: url(images/better.jpg) no-repeat center center fixed;
+    }
+
+    html {
+        scroll-behavior: smooth;
+    }
+    </style>
```

Comment:
Just like it is preferable to have your JS in a separate file, its also preferable to have the CSS in a separate file. Having it in one place like this is preferable to using the style attribute, but even more preferable is having a separate file. (Or just adding to the style.css sheet you already have for the universal stuff)

For stuff that changes on a per-page basis (lets say you had a different background image for each page), you could either 1. If there is enough stuff to be worth it, make an about.css file (and one for each relevant HTML file) 2. If background is the one thing you want to change per-page you could add a class to body with a different class for each page. Say, <body class="about> for one page and <body class="images"> on another, and your master CSS sheet can have different rules to target those classes